docs/tutorials/remote_client.md

```diff
@@ -54,24 +61,28 @@ access to the remote system.  This limitation is being worked on.
 ### Remote node setup
 
 #### Initiate an ssh session to the Podman node
-To use the remote client, we must establish an ssh connection to the Podman server.  We will also use
-that session to bind the remote varlink socket locally.
-
-```
-$ ssh -L 127.0.0.1:1234:/run/podman/io.podman root@remotehost
-```
-Note here we are binding the Podman socket to a local TCP socket on port 1234.
-
-#### Running the remote client
-With the ssh session established, we can now run the remote client in a different terminal window. You
-must inform Podman where to look for the bound socket you created in the previous step using an
-environment variable.
+To use the remote client, we must establish an ssh connection to the Podman server.
+Using the varlink bridge, we're going to initiate a ssh tunnel to connect to the server. You
+must inform Podman where to look for the sshd service on which server.
```

Comment:
I'm not a fan of the words 'we' or 'you' in a man page.  Suggest:

To use the remote client, an ssh connection to the Podman server must be established.
Using the varlink bridge, an ssh tunnel must be initiated to connect to the server.  Podman must
then be informed of the location of the sshd server on the targeted server.